src/coreclr/jit/emit.h

```diff
@@ -722,10 +728,13 @@ class emitter
         opSize   _idOpSize : 3;   // operand size: 0=1 , 1=2 , 2=4 , 3=8, 4=16, 5=32
                                   // At this point we have fully consumed first DWORD so that next field
                                   // doesn't cross a byte boundary.
-#elif defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
+#elif defined(TARGET_ARM64)
+        opSize  _idOpSize : 3; // operand size: 0=1 , 1=2 , 2=4 , 3=8, 4=16
+        insOpts _idInsOpt : 6; // options for instructions
+#elif defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
 /* _idOpSize defined below. */
 #else
-        opSize _idOpSize : 2; // operand size: 0=1 , 1=2 , 2=4 , 3=8
+        opSize    _idOpSize : 2; // operand size: 0=1 , 1=2 , 2=4 , 3=8
```

Comment:
I assume this space is so that it matches the generic version on line 722/728. But, the Arm64 versions on 732 and 733 don't match.

Reply:
May be I am not getting the line numbers correctly so not following your question. But these 2 fields were moved higher up to compact the data within byte boundary. If I don't do this, `_idSmallCns` was overflowing to `16th` byte.

Reply:
This was just formatting nit. I meant the literal space characters between "opSize" and "_idOpSize"